robots/pkg/kubevirt/cmd/get/get.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
The format of this GPL header looks different than in other files, is that in purpose?

Reply:
I guess that is copy paste from an older file. I always try to make the headers look the same, this one I overlooked. Will update, thanks!

Reply:
Updated, PTAL, thank you!